src/manifest.json

```diff
@@ -24,12 +24,12 @@
   },
   "browser_action": {
     "default_icon": {
-      "32": "assets/icon-32.png",
-      "48": "assets/icon-48.png",
-      "64": "assets/icon-64.png",
-      "96": "assets/icon-96.png"
+      "32": "assets/badge-indicator-off.svg",
+      "48": "assets/badge-indicator-off.svg",
+      "64": "assets/badge-indicator-off.svg",
+      "96": "assets/badge-indicator-off.svg"
```

Comment:
If the `browser_action` has an SVG icon, one can just

```
"default_icon": "assets/badge-indicator-off.svg",
```

according to [the docs](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/browser_action). Interestingly, this does not work for the extension icon.

Reply:
Good catch and now I'm curious why you can't use just one for `icons`.